sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -121,7 +109,36 @@ class CommunicationIdentifierKind(with_metaclass(_CaseInsensitiveEnumMeta, str,
     """Communication Identifier Kind.
     """
     Unknown = "UNKNOWN"
-    CommunicationUser = "COMMUNICATIONuSER"
-    PhoneNumber = "PHONEnUMBER"
+    CommunicationUser = "COMMUNICATIONUSER"
+    PhoneNumber = "PHONENUMBER"
     CallingApplication = "CALLINGAPPLICATION"
-    MicrosoftTeamsUser = "MICROSOFTTEAMSuSER"
+    MicrosoftTeamsUser = "MICROSOFTTEAMSUSER"
+
+class CommunicationCloudEnvironment(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
+    """
+    The cloud enviornment that the identifier belongs to
+    """
+    
+    Public = "PUBLIC"
+    Dod = "DOD"
+    Gcch = "GCCH"
+
+class MicrosoftTeamsUserIdentifier(object):
+    """
+    Represents an identifier for a Microsoft Teams user.
+    :ivar user_id: the string identifier representing the identity
+    :vartype user_id: str
+    :param user_id: Value to initialize MicrosoftTeamsUserIdentifier.
+    :type user_id: str
+    :ivar cloud: Cloud environment that this identifier belongs to
+    :vartype cloud: CommunicationCloudEnvironment
+    :ivar is_anonymous: set this to true if the user is anonymous for example when joining a meeting with a share link
+    :vartype is_anonymous: bool
+    :param is_anonymous: Value to initialize MicrosoftTeamsUserIdentifier.
+    :type is_anonymous: bool
+    """
+    def __init__(self, user_id, identifier=None, cloud=CommunicationCloudEnvironment.Public, is_anonymous=False):
```

Comment:
Are you missing `identifier` in the metadata above?

Let's copy over the exact wording from C# to disambiguate the different kind of ids:

identifier: "The full id of the Microsoft Teams User identifier."
user_id: "The id of the Microsoft Teams user. If the user isn't anonymous, the id is the AAD object id of the user."

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Common/src/MicrosoftTeamsUserIdentifier.cs